pkg/kubelet/util/sliceutils/sliceutils.go

```diff
@@ -53,6 +53,9 @@ func (s PodsByCreationTime) Less(i, j int) bool {
 type ByImageSize []kubecontainer.Image
 
 func (a ByImageSize) Less(i, j int) bool {
+	if a[i].Size == a[j].Size {
```

Comment:
just curious, why is this needed?

Reply:
Oh it seems to be a mistake in the merging process. Thanks for pointing it out.